.idea/codeStyles/codeStyleConfig.xml

```diff
@@ -0,0 +1,5 @@
+<component name="ProjectCodeStyleConfiguration">
+  <state>
+    <option name="USE_PER_PROJECT_SETTINGS" value="true" />
```

Comment:
unless it is quite helpful, maybe can remove .idea folder from pr and add to .gitignore